### PR TITLE
Updates to UG

### PR DESCRIPTION
**Pull request by Preshita01**
I also went through the whole UG and edited some parts of it to make it more user friendly.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -300,35 +355,34 @@ Examples:
 
 ### Adding a lesson: `add -l`
 
-Adds a new lesson to TutorAid.
+Adds a new lesson to TutorAid. The lesson's capacity, price and timing are optional details for tutors to include.
```
Comment (JunWei3112) [inline docs/UserGuide.md:358]: Just a minor nitpick here! Can considering changing from `tutors` to `you` to make it more personalised

File: docs/UserGuide.md [hunk 2]
```diff
@@ -300,35 +355,34 @@ Examples:
...
 
 Format: `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIMING]`
 
 * The lesson name should only contain alphanumeric characters and spaces.
 * If provided, the lesson's capacity must be a **positive integer** 1, 2, 3, …
-* If provided, the lesson's price must be a **non-negative number** with either 0 or 2 decimal places. Examples of a valid price are `80` and `85.50`.
+* If provided, the lesson's price must be a **non-negative number** with either 0 or 2 decimal places. Examples of a valid price are `80` and `85.50`. 
+* The lesson's capacity, price and timing are optional details for tutors to include.
```
Comment (JunWei3112) [inline docs/UserGuide.md:365]: Can consider removing this sentence since it is already mentioned

File: docs/UserGuide.md [hunk 3]
```diff
@@ -300,35 +355,34 @@ Examples:
...
 Format: `del -l LESSON_INDEX`
 
 * Deletes the lesson at the specified `LESSON_INDEX`.
-* `LESSON_INDEX` refers to the index number shown in the displayed lesson list.
+* `LESSON_INDEX` refers to the index number shown in the lesson panel.
 * `LESSON_INDEX` must be a **positive integer** 1,2,3, …​
 
 Examples:
-* `del -l 3`
+* `del -l 3` deletes the 3rd lesson displayed in the lesson panel from TutorAid
```
Comment (JunWei3112) [inline docs/UserGuide.md:381]: A very very minor nitpick here, but can consider adding a period `.` at the end at the sentence for consistency!

**Review by JunWei3112 — COMMENTED** [review]
Just some minor issues to be fixed. Else, LGTM!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -45,70 +51,83 @@ Commands in these highlighted boxes are typically used to show what you should e
...
 #### Menu Bar
+
 This is an area where some features can be found. As TutorAid primarily interacts with you through the command box, this area is infrequently used.
 
 #### Command Box
+
 The Command Box is a field in which you can type instructions (commands) to TutorAid. Text in this user guide that is `highlighted` should typically be copied into the command box exactly (including spaces). After you have typed a command into this box, you can press ENTER :leftwards_arrow_with_hook: on your keyboard to tell TutorAid to execute your command.
 
-If your command is of the wrong [format](#23-command-format), TutorAid will try to show you the correct format using the Console if it is able to guess what command you had intended to provide.
+If your command is of the wrong [format](#2.3-command-format), TutorAid will try to show you the correct format in the Console if it is able to infer what command you had intended to provide.
```
Comment (wpinrui) [inline docs/UserGuide.md:66]: The 23 wasn't a typo - the hotlink omits the dot. Can you confirm this link still works?

**Review by wpinrui — APPROVED** [review]
LGTM after merge conflicts are fixed

File: docs/UserGuide.md (hunk 4 above)
Comment (Preshita01) [reply docs/UserGuide.md]: Right, I read up a bit on it and realised it is better to leave it as `23` as well, thank you for highlighting!

**Review by JunWei3112 — APPROVED** [review]
LGTM!

**Pull request merged**